sonar-text-dotnet/pipeline_scripts/execute_tests.ps1

```diff
@@ -0,0 +1,24 @@
+$signAssembly = "$env:CIRRUS_BRANCH" -eq "master" -or "$env:CIRRUS_BRANCH".startsWith("branch-")
+Write-Host "Should the assembly be signed: $signAssembly"
+
+Write-Host "Executing unit tests"
+if($signAssembly)
```

Comment:
AltCoverStrongNameKey should be only set if the DLLs are signed.

Reply:
Was the build failing because I forgot to set `AltCoverStrongNameKey` property?

Reply:
Yes, the same thing that is happening currently on master was happening.